reliontomo/protocols/protocol_matching_coordinates.py

```diff
@@ -0,0 +1,95 @@
+# **************************************************************************
+# *
+# * Authors: Yunior C. Fonseca Reyna    (cfonseca@cnb.csic.es)
+# *
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+from enum import Enum
+from pyworkflow import BETA
+from pyworkflow.protocol.params import (PointerParam)
+from pwem.protocols import EMProtocol
+from reliontomo.objects import RelionSetOfPseudoSubtomograms
+
+from tomo.protocols import ProtTomoBase
+from tomo.objects import SetOfSubTomograms
+
+
+class outputObjects(Enum):
+    relionParticles = RelionSetOfPseudoSubtomograms
+
+class MatchingCoordinates(EMProtocol, ProtTomoBase):
```

Comment:
Could you please rename this protocol to something more specific:
RelionTomoCoordinatesMatcher ?